server/bounding_boxes.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+import gradio as gr
+from PIL import Image
+from PIL import ImageDraw
+import json
+from pathlib import Path
+
+from utils import raise_error
+
+logger = logging.getLogger(__name__)
+
+RESOLUTION_DIR = Path("/home/apex/apex-ocr-common/apex_ocr/data/")
```

Comment:
Same comment as resolution dir in the engine. Move to global as config or something